### PR TITLE
Fix errors in 005

### PR DESCRIPTION
**Pull request by MissJae**
Fixed error on line 9 of the ex.ml file. 
Made changes to the dune file.
@cuihtlauac

### CODE REVIEW

**Comment by cuihtlauac** [comment]
This is correct, but your PR have conflicts to resolve.

**Review by cuihtlauac — CHANGES_REQUESTED** [review]
Resolve conflicts

**Comment by MissJae** [comment]
> Resolve conflicts

I've resolved the conflicts.

File: practice/005/ex.ml [hunk 1]
```diff
@@ -6,7 +6,7 @@ end
 
 module Make(Tested: Testable) : sig val v : test end = struct
   let v = "rev" >::: [
-    "nil" >:: (fun _ -> assert_equal None (Tested.rev []));
+    "nil" >:: (fun _ -> assert_equal [] (Tested.rev []));
```
Comment (cuihtlauac) [inline practice/005/ex.ml]:
```suggestion
    "nil" >:: (fun _ -> assert_equal [] (Tested.rev []));
```

**Review by cuihtlauac — CHANGES_REQUESTED** [review]

**Review by cuihtlauac — APPROVED** [review]
Thanks @MissJae, I had overlooked that issue, it's nice you noticed and fixed it.

**Comment by MissJae** [comment]
> Thanks @MissJae, I had overlooked that issue, it's nice you noticed and fixed it.

Thanks @cuihtlauac

**Pull request merged**